code/datums/components/crafting/recipes.dm

```diff
@@ -401,6 +401,25 @@
 	time = 100
 	category = CAT_MISC
 
+/datum/crafting_recipe/motorized_wheelchair
+	name = "Motorized Wheelchair"
+	result = /obj/vehicle/ridden/wheelchair/motorized
+	reqs = list(/obj/item/stack/sheet/metal = 10,
+		/obj/item/stack/rods = 8,
+		/obj/item/stock_parts/manipulator = 2,
+		/obj/item/stock_parts/capacitor = 1)
+	blacklist = list(
+		/obj/item/stock_parts/manipulator/femto,
+		/obj/item/stock_parts/manipulator/pico,
+		/obj/item/stock_parts/manipulator/nano,
+		/obj/item/stock_parts/capacitor/quadratic,
+		/obj/item/stock_parts/capacitor/super,
+		/obj/item/stock_parts/capacitor/adv
```

Comment:
So if I want a motorized wheelchair with upgraded parts, I have to grab NORMAL parts, make the wheelchair, then replace those parts with upgraded ones? Why not just remove the stock parts from the recipe and have the motorized wheelchair come "empty" by default?

Reply:
I made it so it adds the parts used in the crafting recipe.